src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/barcode/BarcodeEventRepository.kt

```diff
@@ -17,8 +18,9 @@ import javax.validation.constraints.NotNull
 
 @Repository
 interface BarcodeEventRepository : JpaRepository<BarcodeEvent, Long> {
-  fun findByBarcode(barcode: Barcode): List<BarcodeEvent>
   fun findByBarcodeAndStatusOrderByCreatedDateTime(barcode: Barcode, status: BarcodeStatus): List<BarcodeEvent>
+  @Query("select be from BarcodeEvent as be where be.barcode = :barcode and be.status = 'CREATED'")
```

Comment:
(very) Minor, but can we introduce blank lines between methods, especially those with annotations, as it isn't immediately clear which method the `@Query` belongs to